### DISCUSSION TITLE
Change commit convention

### DISCUSSION THREAD

**Discussion by oSumAtrIX** [ReVanced/revanced-discussions#996] [General]
# About 

Previously we were using the convention to commit everything lowercase. Since patches have been renamed, we now use friendly names in the scope. Example: `feat(YouTube - Some Patch): add the ability to do something`. This was chosen arbitrarily and may be revised.

# Solution

Use the following convention: `feat(YouTube - Some Patch): Add the ability to do something`. 
Notice how the casing after `:` changed.

# Motivation

The changelogs currently look like this: 

![image](https://github.com/ReVanced/revanced-discussions/assets/13122796/b771dc10-9f71-49d9-8f4a-d580e9754fc7)

Instead, they could look like this:

![image](https://github.com/ReVanced/revanced-discussions/assets/13122796/6e7e76b9-7f8a-41ce-a221-660b8595f7a3)

# Additional context

It may also be a good idea to look into the scope formatting. Using `-` looks odd. I'd be using `:` instead but then the changelogs would look like this:

![image](https://github.com/ReVanced/revanced-discussions/assets/13122796/6469ad34-03c4-4b08-9c58-152aa0d4dc60)

@ReVanced/revanced-patches

**Comment by LisoUseInAIKyrios** [comment]
More readable is always good.

I think using the existing dash between the product and feature looks better

YouTube - Theme : New and exciting stuff
